src/demo-app/demo-app.scss

```diff
@@ -31,8 +29,25 @@
     font-size: 20px;
   }
 
-}
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
All the styles that apply directly `demo-root` should go right under the definition, before the nested selectors.